landoapi/api/landings.py

```diff
@@ -118,6 +121,13 @@ def update(landing_id, data):
         revision (sha) of push if landed == true
         empty string if landed == false
     """
+    if request.headers['API-Key'] != TRANSPLANT_API_KEY:
```

Comment:
Nice read I found with example code, didn't really know about this stuff before: https://codahale.com/a-lesson-in-timing-attacks/.